gnomad_qc/v5/sample_qc/relatedness.py

```diff
@@ -130,10 +146,18 @@ def main(args):
                 parquet_uri=get_cuking_input_path(test=test),
                 overwrite=overwrite,
             )
+        if args.create_cuking_relatedness_table:
+            logger.info("Converting cuKING outputs to Hail Table.")
```

Comment:
nitpick question, does our team usually use a single period here or an ellipsis?

Reply:
Ellipsis, I can change this